plugins/spi/SPIOutput.cpp

```diff
@@ -271,7 +271,8 @@ SPIOutput::SPIOutput(const UID &uid, SPIBackendInterface *backend,
       personalities.begin() + PERS_WS2812B_COMBINED - 1,
       Personality(WS2812B_SLOTS_PER_PIXEL,
                   "WS2812b Combined Control",
-                  sdc_irgb_combined));
+                  sdc_rgb_combined));
+
```

Comment:
Can we remove the extra blank line please.

Reply:
Or was the linter complaining?

Reply:
Don't think the linter was complaining, didn't really think about it. I can see if I can take it out.